sycl/test-e2e/ClusterLaunch/cluster_launch_enqueue_functions.cpp

```diff
@@ -0,0 +1,59 @@
+// REQUIRES: cuda
+// RUN: %{build} -Xsycl-target-backend --cuda-gpu-arch=sm_90 -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/ext/oneapi/experimental/cluster_group_prop.hpp>
+#include <sycl/ext/oneapi/experimental/enqueue_functions.hpp>
```

Comment:
Why do you want to use this with the enqueue_functions? Can you not just test one thing at a time?

Reply:
So Idea is to expose all launches with properties via the enqueue functions extensions only. Though yes, we have a parallel for overload accepting the property list, which came in the implementation of the extension, but from what I gather, that's not the intended way to launch a kernel with property, and hence not documented anywhere. That's also why the cluster group spec talks about the launch via a launch property only,

hence enqueue functions

Reply:
OK sure. If you should also be able to do this with the `parallel_for` overload, then I would also add this to this test